bindings/node/builtin.js

```diff
@@ -1,7 +1,25 @@
+const DRAFT4_TO_DRAFT6 = require('../../rules/jsonschema-draft4-to-draft6.json')
+const DRAFT6_TO_DRAFT7 = require('../../rules/jsonschema-draft6-to-draft7.json')
+const DRAFT7_TO_V2019_09 = require('../../rules/jsonschema-draft7-to-2019-09.json')
+const V2019_09_TO_V2020_12 = require('../../rules/jsonschema-2019-09-to-2020-12.json')
+
+// TODO: Find a way to specify transitiveness in a less verbose manner
 module.exports = {
   jsonschema: {
+    draft4: {
+      draft6: [DRAFT4_TO_DRAFT6],
+      draft7: [DRAFT4_TO_DRAFT6, DRAFT6_TO_DRAFT7],
+      '2019-09': [DRAFT4_TO_DRAFT6, DRAFT6_TO_DRAFT7, DRAFT7_TO_V2019_09],
+      '2020-12': [DRAFT4_TO_DRAFT6, DRAFT6_TO_DRAFT7, DRAFT7_TO_V2019_09, V2019_09_TO_V2020_12]
+    },
+    draft6: {
+      draft7: [DRAFT6_TO_DRAFT7]
+    },
+    draft7: {
+      '2019-09': [DRAFT7_TO_V2019_09]
+    },
     '2019-09': {
-      '2020-12': require('../../rules/jsonschema-2019-09-to-2020-12.json')
+      '2020-12': [V2019_09_TO_V2020_12]
```

Comment:
I had tried doing this expansion as well, but landed on having multilpe "from" drafts and multiple "to" drafts.  I figured it's easier to define a 4 -> 2019-09 migration directly than to go through 6 and 7 first.

I'm not sure if that concept fits with your architecture, though.

Reply:
That would work, but it would lead to some duplication of rules given the current architecture. I'll take a look at this later. We should be able to concisely define the ordering of versions and have the code figure it out